File: cpp-src/basic/src/apps/ServerApp.cpp

#include "socket/server.hpp"

/**
 * @brief basic starting point 
 *
 *      Author: gash
 */
int main(int argc, char **argv) {
    //basic::BasicServer javaSvr("127.0.0.1", 3000);
    //javaSvr.start();

    basic::BasicServer cppSvr("127.0.0.1", 2000);
    cppSvr.start();
    
    basic::BasicServer pythonSvr("127.0.0.1", 4000);
    pythonSvr.start();

    //basic::BasicServer cppSvr("127.0.0.1", 2000);
    //cppSvr.start();
    
}

File: java-src/basic/src/gash/app/ServerApp.java
package gash.app;

import gash.socket.BasicServer;

/**
 * server application
 * 
 * @author gash
 * 
 */
class ServerApp {
	public ServerApp() {
	}

	public static void javaServer(String[] args) {
		var host = "127.0.0.1";
		var port = 3000;
		var server = new BasicServer(host,port);
		server.start();
	}

	public static void main(String[] args) {
		javaServer(args);
	}
}
